helix-rest/src/test/java/org/apache/helix/rest/server/TestInstancesAccessor.java

```diff
@@ -159,8 +159,12 @@ public void testGetAllInstances() throws IOException {
 
     Set<String> instances = OBJECT_MAPPER.readValue(instancesStr,
         OBJECT_MAPPER.getTypeFactory().constructCollectionType(Set.class, String.class));
-    Assert.assertEquals(instances, _instancesMap.get(CLUSTER_NAME), "Instances from response: "
-        + instances + " vs instances actually: " + _instancesMap.get(CLUSTER_NAME));
+    Assert.assertEquals(instances.size(), _instancesMap.get(CLUSTER_NAME).size(), "Different amount of elements in "
+        + "the sets: " + instances.size() + " vs: " + _instancesMap.get(CLUSTER_NAME).size());
+    Assert.assertTrue(instances.containsAll(_instancesMap.get(CLUSTER_NAME)), "instances set does not contain all "
+        + "elenets of _instanceMap");
```

Comment:
Hi, your fix looks good. Could you make sure to check the spelling in your fix? For instance, please correct "elenets" to "elements" here.